android/src/main/java/com/urbanairship/reactnative/EventEmitter.java

```diff
@@ -140,7 +139,7 @@ private boolean emit(final Event event) {
         try {
             reactContext.getJSModule(RCTNativeAppEventEmitter.class).emit(event.getName(), event.getBody());
         } catch (Exception e) {
-            Log.d("UrbanAirshipReactModule", "Failed to emit event", e);
```

Comment:
We use Log in some places, Logger in others. Logger is internal only so it was giving us a warning. That is why I introduced the PluginLogger like we have on Cordova. We should make it easier to create Loggers going forward.